src/bootstrap/native.rs

```diff
@@ -607,7 +607,6 @@ impl Step for Sanitizers {
         cfg.define("COMPILER_RT_BUILD_SANITIZERS", "ON");
         cfg.define("COMPILER_RT_BUILD_XRAY", "OFF");
         cfg.define("COMPILER_RT_DEFAULT_TARGET_ONLY", "ON");
-        cfg.define("COMPILER_RT_USE_LIBCXX", "OFF");
```

Comment:
The `COMPILER_RT_USE_LIBCXX` controls if compiler-rt should build libc++ from
the source tree when present. In the context of sanitizers in particular, it
means that custom instrumented variants of the C++ standard library will be
built.

There should be no need to do any of that, so option should remained turned off.

Reply:
Thanks for the insights!